index.html

```diff
@@ -2309,6 +2309,112 @@ <h2 id="AudioDestinationNode">
               not be changed.
             </p>
           </dd>
+          <dt>
+            readonly attribute double latency
+          </dt>
+          <dd>
+            <p>
+              The estimation in seconds of audio output latency, i.e. the delay
+              between the moment the implementation sends a buffer to the
+              system and the audible sound from the audio output (speaker or
+              connector).
+            </p>
+            <p>
+              The <a><code>latency</code></a> attribute value depends on the
+              platform and the hardware audio output device, the attribute
+              value does not change for the context's lifetime as long as audio
+              output device remains the same. If the audio output device is
+              changed the <a><code>latency</code></a> attribute value will be
+              updated accordingly.
+            </p>
+          </dd>
+          <dt>
+            readonly attribute AudioTimestamp devicePosition
+          </dt>
+          <dd>
+            <p>
+              Holds an <a><code>AudioTimestamp</code></a> instance containing
+              two correlated context's audio stream position values: the
+              <a href=
+              "#widl-AudioTimestamp-contextTime"><code>contextTime</code></a>
+              member contains estimation of the audio stream position which is
+              currently being rendered by the
+              <a><code>AudioDestinationNode</code></a> (i.e. the current output
+              device audio stream position), in the same units and origin as
+              context's <a href=
+              "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>;
+              the <a href=
+              "#widl-AudioTimestamp-performanceTime"><code>performanceTime</code></a>
+              member contains performance time estimation of the moment when
+              the stored audio stream position is rendered, in the same units
+              and origin as <code>performance.now()</code> (described in
+              [[!hr-time-2]]).
+            </p>
+            <p>
+              If the context's rendering graph has not yet processed a block of
+              audio, then <a><code>devicePosition</code></a> holds an
+              <code>AudioTimestamp</code> instance with both members containing
+              zero.
```

Comment:
it's not exactly performance.now(), it's a performance time stamp of the moment when the audible output stream position was equal to `devicePosition.contextTime`. If nothing has been played the output stream position is equal to zero and both `devicePosition` field contain zero as well.